package.json

```diff
@@ -37,6 +37,7 @@
     "babel-plugin-dynamic-import-node": "^1.1.0",
     "babel-runtime": "^6.25.0",
     "clone-deep": "^1.0.0",
+    "date-fns": "^1.29.0",
```

Comment:
Remember to add `yarn.lock` too before merging

Reply:
oh, seems that I have to install yarn (normaly I'm working with npm)

Reply:
Yup, we ran into a lot of issues with npm 5 (see https://github.com/yunity/karrot-frontend/issues/543)